### ISSUE TITLE
Laravel 5.6 no longer supports Optimize

### ISSUE THREAD

**Issue by nemhal** [lorisleiva/laravel-deployer#36]
Hello,

First, great work and nice wrapper for deployer.

Artisan Optimize was removed from Laravel 5.6, so I guess that should be also removed from /config/deployer.php line 64:

```
        // Deployment is done but not live yet (before symlink)
        'ready' => [
            'artisan:storage:link',
            'artisan:view:clear',
            'artisan:cache:clear',
            'artisan:config:cache',

            'artisan:optimize',

            'artisan:migrate',
            'artisan:horizon:terminate',
        ],
```

**Comment by lorisleiva** [comment]
Hi there 👋 

Thank you for pointing this out.

Just so you know, the command `php artisan optimize` will not run if your Laravel version is higher than 5.4 as you can see in the task definition below:

https://github.com/lorisleiva/laravel-deployer/blob/3d168965b385a096fec7fffa3fb9e3fb47adaba2/src/task/artisan.php#L38-L39

However, since Laravel Deployer support Laravel from version 5.5, it is a useless task to add to our deployment flow. I will make a commit to fix this today. 🙂

**Comment by nemhal** [comment]
In Laravel 5.5 artisan optimize will just do nothing since all the code was removed, but from Laravel 5.6 it can break the application since the call to the command was entirely removed: 

https://github.com/laravel/framework/pull/20851/files
https://laravel-news.com/laravel-5-6-removes-artisan-optimize

**Comment by lorisleiva** [comment]
I am aware of that. You must have misunderstood me. Currently if you are using the `artisan:optimize` task within Laravel 5.5 or higher it will simply do nothing, i.e. not run `php artisan optimize`.

**Comment by adevade** [comment]
@lorisleiva Digging up an old issue here instead of creating a new one. 

Since 5.7 the `optimize` command is back, together with `optimize:clear`. I use these on every deployment for caching, and would appreciate to have them in your package. Maybe with `min => 5.7`, seeing as your package still supports 5.5 and up?

Source: https://github.com/laravel/framework/pull/23468

**Comment by lorisleiva** [comment]
@adevade Thanks for letting me know, I had no idea it was back! I'll re-add the `artisan:optimize` task and add the new `artisan:optimize:clear` asap.

**Comment by lorisleiva** [comment]
🚀 Done and released in [`v0.2.14`](https://github.com/lorisleiva/laravel-deployer/releases/tag/v0.2.14).
